src/ledger/parse/ledger.ts

```diff
@@ -67,19 +66,19 @@ function parseState(state) {
  * @throws RangeError: Invalid time value (rippleTimeToISO8601)
  */
 export function parseLedger(ledger: Ledger): FormattedLedger {
-  const ledgerVersion = parseInt(ledger.ledger_index || ledger.seqNum, 10)
```

Comment:
Any particular reason why we're using `parseInt(..., 10)`, could we just use Number(...) if parsing in base 10?

Reply:
No reason. I suspect `Number()` would work equally well, but I'm not 100% sure. It has been `parseInt()` from [the beginning of time](https://github.com/ripple/ripple-lib/pull/504/files#diff-4f915b23ac0d38c14331690bd88e1b4af5588dacf2f87fd658748bebb21742f1R39) and I think we can leave it that way unless there is some reason to change it.

Reply:
I'm perfectly happy with `parseInt()`, mostly just curious.